codegen.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  schema: [
+    {
+      "http://localhost:9090/graphql/query": {
+        headers: {
+          cookie: process.env.REACT_APP_GQL_COOKIE,
+        },
+      },
+    },
+  ],
+  documents: ["./src/**/*.ts"],
+  overwrite: true,
+  generates: {
+    "./src/gql/generated/types.ts": {
+      plugins: ["typescript", "typescript-operations"],
+      config: {
+        skipTypename: true,
```

Comment:
idk if we want to preserve typenames but it produces much cleaner types by skipping it. But it can be useful to have it if want to perform operations based off the corresponding type  like we do in `src/pages/task/logs/LogTypes.tsx`